shapes/contact/person.shex

```diff
@@ -22,8 +34,16 @@ vcard:Individual {
     vcard:role xsd:string ? ;
     vcard:title xsd:string ? ;
     vcard:note xsd:string ? ;
+    vcard:inAddressBook vcard:AddressBook + ;
+    # vcard:inAddressBook is a custom term
+    # vcard:AddressBook is a custom class
 }
 
+# We store the preferred WebID for each person. The use of vcard:WebId
+# refer to the Individual's WebIDs, but the idea is that we could expand it
+# to refer to other type of unique identifiers later
+# vcard:WebId is a custom term
+# Big question - SHOULD something as fundamental as WebID be defined in vCard?
 vcard:WebId {
     vcard:value xsd:string {1};
```

Comment:
```suggestion
    vcard:value xsd:anyURI * ;
```

Why would a person have one and only one WebID?
I would be very enclined to be more lenient here. And more strict on the type.

As mentioned elsewhere, I don't think we should have a vcard:WebID class and we should have a vcard:hasWebID property.

Reply:
Yeah, that's certainly a good point (i.e. a Person being limited to just one WebID). I created a separate issue just for that question: https://github.com/inrupt/pod-browser/issues/251